### PR TITLE
Add get contributions contract

### PR DESCRIPTION
**Pull request by PierreOucif**

### CODE REVIEW

File: marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml [hunk 1]
```diff
@@ -5186,6 +5186,80 @@ paths:
...
+        - bearerAuth: [ ]
+      tags:
+        - ReadContributions
+      summary: Get all contributions
+      operationId: getContributions
+      description: |
+        Get contributions sort by date.
+      parameters:
+        - in: query
+          name: projectId
```
Comment (AnthonyBuisset) [inline marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml:5201]: use array as well
You also need projectSlugs as most url have only the slug in input

File: marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml [hunk 2]
```diff
@@ -5186,6 +5186,80 @@ paths:
...
+              $ref: '#/components/schemas/GithubUserId'
+        - in: query
+          name: projectContributorLabelIds
+          description: Contributor label filter
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/ProjectContributorLabelId'
+        - in: query
+          name: hasBeenRewarded
```
Comment (AnthonyBuisset) [inline marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml:5233]: reward amount ?

File: marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml [hunk 3]
```diff
@@ -11517,4 +11604,59 @@ components:
...
+              $ref: '#/components/schemas/ContributionActivityStatus'
+            contributors:
+              type: array
+              items:
+                $ref: '#/components/schemas/ContributorResponse'
+            applicants:
+              type: array
+              items:
+                $ref: '#/components/schemas/ContributorResponse'
+            issuesLinked:
```
Comment (AnthonyBuisset) [inline marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml:11658]: linkedIssues

**Review by AnthonyBuisset — APPROVED** [review]

File: marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml [hunk 4]
```diff
@@ -5186,6 +5186,88 @@ paths:
...
+              $ref: '#/components/schemas/ProjectSlug'
+        - in: query
+          name: statuses
+          description: Contribution activity status filter
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/ContributionActivityStatus'
+        - in: query
+          name: repositoryIds
```
Comment (ofux) [inline marketplace-api-contract/src/main/resources/marketplace-api-contract.yaml:5220]: Pour garder la convention de nomage qu'on a, je mettrais `repoIds`

**Review by ofux — APPROVED** [review]

**Pull request merged**